packages/nextra-theme-docs/src/components/sidebar.tsx

```diff
@@ -41,11 +41,11 @@ const classes = {
     '_bg-primary-100 _font-semibold _text-primary-800 dark:_bg-primary-400/10 dark:_text-primary-600',
     'contrast-more:_border-primary-500 contrast-more:dark:_border-primary-500'
   ),
-  list: cn('_flex _flex-col _gap-1'),
+  list: cn('_grid _gap-1'),
   border: cn(
     '_relative before:_absolute before:_inset-y-1',
     'before:_w-px before:_bg-gray-200 before:_content-[""] dark:before:_bg-neutral-800',
-    '_ps-3 before:_start-0'
+    '_ps-3 before:_start-0 _pt-1 _ms-3'
```

Comment:
instead of your `_mt-1`, let's use `_pt-1` which we already used here 
<img width="581" alt="image" src="https://github.com/user-attachments/assets/058c6fc9-47e4-45b2-9ab6-a22c41589d97">